src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -59,17 +59,18 @@ def __init__(self, job_id, initial_params, area, should_export_plots):
             "percentage_completed": 0
         }
         self.should_export_plots = should_export_plots
-        self.market_unmatched_loads = MarketUnmatchedLoads(self.area_result_dict)
-        self.price_energy_day = MarketPriceEnergyDay(should_export_plots)
         self.market_bills = MarketEnergyBills()
-        self.cumulative_bills = CumulativeBills()
-        if ConstSettings.BalancingSettings.ENABLE_BALANCING_MARKET:
-            self.balancing_bills = MarketEnergyBills(is_spot_market=False)
-        self.cumulative_grid_trades = CumulativeGridTrades()
-        self.device_statistics = DeviceStatistics(should_export_plots)
-        self.trade_profile = EnergyTradeProfile(should_export_plots)
         self.kpi = KPI(should_export_plots)
```

Comment:
Just for my understanding: We need kpi and market_bills for the external Strategies (market_info and market_stats) right?

Reply:
Exactly, ideally it could be done on d3a-web too and when the external user needs it, it could be read from DB and sent via d3a-web that way we can further optimize d3a and release resources from d3a to even calculate these results